python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -83,8 +83,15 @@ def __init__(
         self._ray_remote_args = self._apply_default_remote_args(self._ray_remote_args)
         self._ray_actor_task_remote_args = {}
         actor_task_errors = DataContext.get_current().actor_task_retry_on_errors
+        # Ray typically automatically retries system errors. However, in some cases, Ray
```

Comment:
let's add a TODO here. eventually I think we should fix this in Ray Core cc @jjyao